rp2040-hal/src/usb.rs

```diff
@@ -525,11 +544,13 @@ impl UsbBusTrait for UsbBus {
                     }
                 }
             }
-            // check for setup request
-            // Only report setup if OUT has been cleared.
-            if sie_status.setup_rec().bit_is_set() {
-                ep_setup |= 1;
-                inner.read_setup = true;
+            if sie_status_flags.bus_reset().bit_is_set() {
+                // Clear interrupt flag for reset
+                inner
+                    .ctrl_reg
+                    .sie_status
+                    .modify(|_, w| w.bus_reset().set_bit());
```

Comment:
This bit is cleared line [433](https://github.com/rp-rs/rp-hal/blob/af862424633107515fc3ac5f2f044474999f7511/rp2040-hal/src/usb.rs#L433)

Reply:
I'll revert this section to how it was previously